.github/scripts/update-platform-executables.js

```diff
@@ -0,0 +1,108 @@
+const core = require('@actions/core');
+const { GitHub } = require('@actions/github');
+const fs = require('fs');
+
+/**
+ * Functionality from svenstaro/upload-release-action.
+ * Link: https://github.com/svenstaro/upload-release-action
+ */
+const updatePlatformExecutables = async () => {
+  try {
+    const requiredParams = ['GITHUB_TOKEN', 'REPO_NAME', 'TAG_NAME', 'ASSET_NAME', 'FILE'];
+    for (const param of requiredParams) {
+      if (!(param in process.env)) {
+        core.setFailed(`Exiting, required param: ${param} is missing`);
+        return;
+      }
+    }
+    const github = new GitHub(process.env.GITHUB_TOKEN);
+    const [owner, repo] = process.env.REPO_NAME
+      ? process.env.REPO_NAME.split('/')
+      : [null, null];
+    const tag = process.env.TAG_NAME;
+    const getReleaseResponse = await github.repos.getReleaseByTag({
+      owner,
+      repo,
+      tag
+    });
+
+    const {
+      data: {
+        id: oldReleaseId,
+        html_url: oldHtmlUrl,
+        upload_url: oldUploadUrl,
+        body: oldBody,
+        draft: oldDraft,
+        name: oldName,
+        prerelease: oldPrerelease
+      }
+    } = getReleaseResponse;
+
+    core.info(
+      `Got release info: '${oldReleaseId}', ${oldName}, '${oldHtmlUrl}', '${oldUploadUrl},'`
+    )
+    core.info(`Body: ${oldBody}`)
+    core.info(`Draft: ${oldDraft}, Prerelease: ${oldPrerelease}`)
+
+    const assetName = process.env.ASSET_NAME;
+    const file = process.env.FILE;
+    const overwrite = process.env.OVERWRITE;
+
+    const stat = fs.statSync(file);
+    if (!stat.isFile()) {
+      core.setFailed(`Skipping ${file}, since its not a file`);
+      return;
+    }
+    const file_size = stat.size;
+    const file_bytes = fs.readFileSync(file);
+
+    // Check for duplicates.
+    const assets = await github.repos.listAssetsForRelease(
```

Comment:
Is this check to verify the fact that there can be only one release with a given release_id ? 
If yes, let's say there's an asset already present with the same release_id (assuming release Id will have the tags to uniquely identify), what would be the next step? Looks like it'll still go to line #87 ?

Reply:
It will get the list of assets attached to the given release id. [Here](https://github.com/twilio/twilio-cli/pull/299/files#diff-32247ddfd22ce9bef49756459e03737402a5c1fd561e9413e745219398264b1dR66), we are checking for whether the asset is already present with the same name or not. If asset is there with the same name and we don't pass any overwrite flag it will [exit](https://github.com/twilio/twilio-cli/pull/299/files#diff-32247ddfd22ce9bef49756459e03737402a5c1fd561e9413e745219398264b1dR79).